solidity/dashboard/src/connectors/explorerModeSubprovider.js

```diff
@@ -0,0 +1,22 @@
+import { Subprovider } from "@0x/subproviders";
+
+
+class ExplorerModeSubprovider extends Subprovider {
+  constructor(_eventEmitter) {
+    super()
+    this.eventEmitter = _eventEmitter
+  }
+  async handleRequest(payload, next, end) {
+    switch (payload.method) {
+      case "eth_sendTransaction":
+        this.eventEmitter.emit("chooseWallet", payload)
+        next()
```

Comment:
It should be `end()`, shouldn't it? This type of eth method has been already handled by emitting an event, so we don't want to pass it to the next subprovider.